examples/all-clusters-app/all-clusters-common/src/ota-requestor-stub.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/AttributeAccessInterface.h>
+#include <app/util/attribute-storage.h>
+#include <platform/OTARequestorInterface.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::OtaSoftwareUpdateRequestor;
+
+namespace {
+
+class OtaSoftwareUpdateRequestorAttrAccess : public AttributeAccessInterface
+{
+public:
+    // Register for the OTA Requestor Cluster on all endpoints.
+    OtaSoftwareUpdateRequestorAttrAccess() :
+        AttributeAccessInterface(Optional<EndpointId>::Missing(), OtaSoftwareUpdateRequestor::Id)
+    {}
+
+    // TODO: Implement Read/Write for OtaSoftwareUpdateRequestorAttrAccess
+    CHIP_ERROR Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
+    CHIP_ERROR Write(const ConcreteDataAttributePath & aPath, AttributeValueDecoder & aDecoder) override;
+};
+
+OtaSoftwareUpdateRequestorAttrAccess gAttrAccess;
+
+CHIP_ERROR OtaSoftwareUpdateRequestorAttrAccess::Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder)
+{
+    return CHIP_NO_ERROR;
```

Comment:
This is going to get treated the same as not having an override, with fallback to trying to read from the attr store, which does not work for lists, which is why cirque is still failing.

What you want to do is to encode an empty list here.  Something like:
```
return aEncoder.Encode(DataModel::List<uint8_t>())
```
should work (the actual template arg does not matter, since there are no entries, and figuring out the "right" one is annoying; I really need to fix https://github.com/project-chip/connectedhomeip/issues/12850 to make this not an issue).